frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -77,10 +84,57 @@ const ProductAnalyticsOnboarding = (): JSX.Element => {
                 teamPropertyToVerify="ingested_event"
                 stepKey={OnboardingStepKey.VERIFY}
             />
+            <OnboardingProductConfiguration
+                stepKey={OnboardingStepKey.PRODUCT_CONFIGURATION}
+                options={[
+                    {
+                        title: 'Autocapture frontend interactions',
+                        description: `If you use our JavaScript or React Native libraries, we'll automagically 
+                            capture frontend interactions like pageviews, clicks, and more. Fine-tune what you 
+                            capture directly in your code snippet.`,
+                        teamProperty: 'autocapture_opt_out',
+                        value: !currentTeam?.autocapture_opt_out,
+                        type: 'toggle',
+                        inverseToggle: true,
+                    },
+                ]}
+            />
         </OnboardingWrapper>
     )
 }
 const SessionReplayOnboarding = (): JSX.Element => {
+    const { featureFlags } = useValues(featureFlagLogic)
+    const configOptions: ProductConfigOption[] = [
+        {
+            type: 'toggle',
+            title: 'Capture console logs',
+            description: `Capture browser console logs as a part of user session recordings. 
```

Comment:
```suggestion
            description: `Capture console logs as a part of user session recordings. 
```

Can drop the browser since Mobile recording will include this too